### PR TITLE
Add space for Rari Capital

### PR DESCRIPTION
**Pull request by davidlucid**
**Add space for Rari Capital**

Can someone please confirm that the displayed RGT voter scores/shares/balances will be the sum of the RGT balances returned by the two strategies (`erc20-balance-of` and `rari-unclaimed-rgt`)? Or should I write a single strategy to combine the two?

Thank you so much!!!
  
Etherscan: https://etherscan.io/address/0xD291E7a03283640FDc51b121aC401383A46cC623
CoinGecko: https://www.coingecko.com/en/coins/rari-governance-token
Website: http://rari.capital/
Twitter: https://twitter.com/RariCapital

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
The space doesn’t load properly with this strategies.
@bonustrack Could you review it? Thank you.

File: spaces/rari/index.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "token": "0xD291E7a03283640FDc51b121aC401383A46cC623",
+  "name": "Rari Capital",
+  "network": "1",
+  "symbol": "RGT",
+  "skin": "rari",
+  "domain": "vote.rari.capital",
```
Comment (gokaiorg) [inline spaces/rari/index.json:7]: To add a custom domain you will have to add your domain in this file https://github.com/snapshot-labs/snapshot-spaces/blob/master/spaces/domains.json

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/rari/index.json (hunk 1 above)
Comment (davidlucid) [reply spaces/rari/index.json:7]: Done!

**Comment by davidlucid** [comment]
@gokaiorg Added to domains.json!

**Comment by gokaiorg** [comment]
We've made a major global update, please pull before commit again your changes. Thank you.

**Comment by davidlucid** [comment]
@gokaiorg I have pulled the latest changes and updated my index.json!

**Comment by davidlucid** [comment]
Link to strategy pull request: https://github.com/snapshot-labs/snapshot.js/pull/25

File: spaces/rari/index.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      "name": "erc20-balance-of",
+      "params": {
+        "address": "0xD291E7a03283640FDc51b121aC401383A46cC623",
+        "symbol": "RGT",
+        "decimals": 18
+      }
+    },
+    {
+      "name": "rari-unclaimed-rgt",
```
Comment (bonustrack) [inline spaces/rari/index.json:17]: @davidlucid Your PR looks good, i would just suggest to use "contract-call" strategy here, i've sent a comment in the strategy PR: https://github.com/snapshot-labs/snapshot.js/pull/25#issuecomment-720234232 let me know if need more info.

**Comment by davidlucid** [comment]
Now using `contract-call`! @bonustrack

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**